script/lint

```diff
@@ -3,10 +3,10 @@
 
 cd "$(dirname "$0")/.."
 
-export files="`git diff upstream/dev... --diff-filter=d --name-only | grep -e '\.py$'`"
-echo "================================================="
-echo "FILES CHANGED (git diff upstream/dev... --diff-filter=d --name-only)"
-echo "================================================="
+export files="$(git diff $(git merge-base upstream/dev HEAD) --diff-filter=d --name-only | grep -e '\.py$')"
+echo '================================================='
+echo 'FILES CHANGED (git diff $(git merge-base upstream/dev HEAD) --diff-filter=d --name-only)'
```

Comment:
This line has gotten so long, should we just change it to `FILES CHANGED`?

Reply:
Now center aligned, let the bikeshedding begin!